content/blogposts/2021/why-were-friends-not-competitors-with-code-hosts.md

```diff
@@ -7,7 +7,7 @@ description: Some people assume that Sourcegraph competes with GitHub, GitLab, B
 publishDate: 2021-04-06T10:00-07:00
 tags: [blog, code search]
 slug: why-we're-friends-with-code-hosts
-socialImage: https://about.sourcegraph.com/blog/sourcegraph-social-img.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/sourcegraph-social-image.png
```

Comment:
Same default social image taken care of in `<Layout />` here

Reply:
Please see: 3ecfe46